tools/emptyDropsWrapper/emptyDropsWrapper.R

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env Rscript
+
+## Filename: emptyDropsWrapper.R
+## Author: Nikolas Barkas
+## Date: Jan 2019
+## Description: Wrapper for running emptyDrops on a sparse matrix, provides
+##    command line arguments for emptyDrops and I/O handling and
+##    error trapping.
+
+## Load optparse
+library('optparse')
+
+## Define helper functions
+
+#' Prints a message to stderr and exits R with error code 1
+#' @param msg message to standard error
+errorExit <- function(msg) {
+    cat(msg,file=stderr());
+    quit(save='no',status=1);
+}
+
+#' Prints a message with cat only if verbose is TRUE
+#' @param ... parameters passed to cat
+catv <- function(...) {
+    if (verbose) {
+        cat(...)
+    }
+}
+
+#' Converts NAs in a vector to FALSE
+#' @param x vector to convert NAs in
+#' @return the x vector with NAs turned to FALSE
+NA2FALSE <- function(x) {
+    x[is.na(x)] <- FALSE;x
+}
+
+## Parse the input arguments
+option_list <- list(
+    make_option(c('-i','--input-rds'),
+                type='character',
+                default=NULL, ## required
+                dest='input_rds',
+                help='input RDS file containing the data matrix in dgCMatrix format gene x droplet orientation'),
+    make_option(c('-o','--output-csv'),
+                type='character',
+                default=NULL, ## required
+                dest='output_csv',
+                help='output CSV file, if it exists the program will abort'),
+    make_option(c('--fdr-cutoff'),
+                type='numeric',
+                default=0.10, ## FDR recommendation from paper
```

Comment:
Which paper are you referring to here? I think it would be useful to include a link if possible

Reply:
I added a link to the pre-print

Reply:
I still don't see a link here @barkasn 🤔